setup.py

```diff
@@ -1,18 +1,26 @@
-# Copyright (c) 2008,2010,2015,2016 MetPy Developers.
+# Copyright (c) 2008,2010,2015,2016,2019 MetPy Developers.
 # Distributed under the terms of the BSD 3-Clause License.
 # SPDX-License-Identifier: BSD-3-Clause
 """Setup script for installing MetPy."""
 
 from __future__ import print_function
 
 from setuptools import find_packages, setup
-import versioneer
 
-ver = versioneer.get_version()
+
+def metpy_version():
```

Comment:
Did you find this necessary or was this more just brought in because other things did it?

Reply:
This function was created to provide to `download_url`, based on what other did to get the version.